src/version.ts

```diff
@@ -1 +1 @@
-export const version = "0.1.8";
+export const version = "0.1.8"; // TODO(@qu4k): ugly
```

Comment:
```suggestion
export const version = "0.1.9"; // TODO(@qu4k): ugly
```

We need to increment this version before the PR is merged

Reply:
I think we should not, as a proper release can happen after we do the complete rewrite; we should keep what works (0.1.8) for now...

Reply:
Fair enough!